README.md

```diff
@@ -21,31 +21,23 @@ Kuzzle features are accessible through a secured API. It can be used through a l
 
 # How to use it
 
-This Back-office is a static front-end application. All calls to Kuzzle are made directly in the browser. **This is why your Kuzzle Proxy IP and Websocket port (default 7512) must be accessible from everywhere if you want to use this Back-office.**
+This Admin Console is a static front-end application. All calls to Kuzzle are made directly in the browser. **This is why your Kuzzle Proxy IP and Websocket port (default 7512) must be accessible from everywhere if you want to use this Admin Console.**
```

Comment:
Outdated. Proposal:
`The Admin Console is a static front-end client, executed by your web browser, and using the Kuzzle Backend API. This means that your Kuzzle Backend IP address and network port (default: 7512) must be reachable from the computer executing the Admin Console.`